lib/ui/text/font_collection.cc

```diff
@@ -174,4 +146,19 @@ void FontCollection::LoadFontFromList(const uint8_t* font_data,
   collection_->ClearFontFamilyCache();
 }
 
+void FontCollection::LoadFontFromListOrThrow(Dart_Handle font_data_handle,
```

Comment:
nit: I think the `OrThrow` naming is a bit confusing. We generally try to avoid throwing Dart exceptions from C++, except in a few special cases. We often try to return a value or call a callback with some value if there's a problem. In this particular case, we seem to only assert that this code is running on the UI thread.

I think I would just drop the`OrThrow` part to avoid the confusion.

Reply:
I've dropped all the OrThrow suffixes.

But just to give a bit of background; I added those when I realised the old Tonic bindings were implicitly adding that assert on all those functions. In that case I felt it was better to explicitly call that out in the naming to not leave it as a surprise for anyone calling those.
In retrospect that is also a good reason why we don't need it for the new bindings where those checks are explicit.